src/Entity/MembreCentre.php

```diff
@@ -117,6 +117,14 @@ public function canManageProfessionals(): bool
             && true === $this->getDroits()[self::TYPEDROIT_GESTION_MEMBRES];
     }
 
+    public function toggleRight(string $right): void
+    {
+        if (!in_array($right, [self::TYPEDROIT_GESTION_BENEFICIAIRES, self::TYPEDROIT_GESTION_MEMBRES])) {
+            return;
+        }
+        $this->droits[$right] = !$this->droits[$right];
```

Comment:
isn't there a check on if array key exists ? If it doesnot, we should add it i guess